src/m365/planner/commands/tenant/tenant-settings-list.spec.ts

```diff
@@ -11,11 +11,12 @@ const command: Command = require('./tenant-settings-list');
 
```

Comment:
Removed this file from the commit since it is not related to the original command

Reply:
True, I noticed I made a typo in the tests of the other command. Thought it wasn't so bad to resolve this all at once. 🙂